go.mod

```diff
@@ -3,16 +3,28 @@ module github.com/ipfs/ipfs-ds-convert
 go 1.12
 
 require (
-	github.com/ipfs/go-datastore v0.4.4
+	github.com/DataDog/zstd v1.4.5 // indirect
```

Comment:
As with https://github.com/ipfs/go-ds-badger2/pull/31#discussion_r494323001 I think we should keep the go.mod file as clean as possible by just updating the dependencies we need.